app/locales/en.json

```diff
@@ -105,17 +105,16 @@
   "Submit report": "Submit report",
   "Statistics": "Statistics",
   "Download the dataset as CSV": "Download the dataset as CSV",
-  "COVID-19 infected": "COVID-19 infected",
-  "people have reported that they have tested positive for COVID-19 and experience symptoms.": "people have reported that they have tested positive for COVID-19 and experience symptoms.",
+  "COVID-19 infected": "COVID-19 infected", "people have reported that they have tested positive for COVID-19 and experience symptoms.": "people have reported that they have tested positive for COVID-19 and experienced symptoms.",
```

Comment:
Looks like this line is missing a linebreak (two translations on one line)

Reply:
Oh, whoops! Good catch.

Reply:
Fixed.